File: Java/326_Power of Three.java
//Question - https://leetcode.com/problems/power-of-three/description/

class Solution {
    public boolean isPowerOfThree(int n) {
        if (n <= 0) {
            return false;
        }
        return Math.pow(3, 19) % n == 0;

    }
}

File: C++/326_Power of Three.cpp
// Question - https://leetcode.com/problems/power-of-three/

#include <bits/stdc++.h>
using namespace std;
class Solution
{
public:
    bool isPowerOfThree(int n)
    {
        if (n <= 0)
        {
            return false;
        }
        while (n > 1)
        {
            if (n % 3 != 0)
            {
                return false;
            }
            n = n / 3;
        }
        return true;
    }
};
